src/applications/disability-benefits/wizard/pages/file-bdd.jsx

```diff
@@ -25,16 +26,13 @@ function alertContent(getPageStateFromPageName, setWizardStatus) {
   const isLastDayToFileBDD = daysRemainingToFileBDD === 0;
   const dateOfLastBDDEligibility = moment()
     .add(daysRemainingToFileBDD, 'days')
-    .format('MMM D, YYYY');
+    .format('MMMM D, YYYY');
```

Comment:
Does this follow the date conventions specified here? https://design.va.gov/content-style-guide/dates-and-numbers

I think this will provide full month name.

Reply:
Yes, we should only abbreviate the month name when space is limited. Using `MMMM` will use the full month name which is recommended.

Reply:
Lol I should have read the guidance more closely 🙈